lib/components/Masthead/Masthead.module.scss

```diff
@@ -245,11 +245,12 @@ $search-max-size: 100 * $grid-size;
                 }
 
                 &.user-btn {
-                    >:global(.icon-alias-user) {
-                        width: $thumbnail-size;
-                        height: $thumbnail-size;
-                        font-size: $font-size-h5;
-                        border-radius: 50%;
+                    width: $thumbnail-size;
+                    height: $thumbnail-size;
+                    border-radius: 100%;

```

Comment:
nit: 50% border-radius, with 100% the browser has to do additional scaling to get the circle

Reply:
done